src/PointwiseFunctions/Elasticity/ConstitutiveRelations/CubicCrystal.hpp

```diff
@@ -0,0 +1,138 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+#include <pup.h>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace Elasticity::ConstitutiveRelations {
+
+/*!
+ * \brief A cubic crystalline material
+ *
+ * \details For a cubic crystalline material the Elasticity tensor in the linear
+ * constitutive relation \f$T^{ij}=-Y^{ijkl}S_{kl}\f$ reduces to
+ *
+ * \f[
+ * Y^{ijkl} = \frac{1 - \nu}{\nu}\lambda \delta^{ij}\delta^{kl}\delta^{jk} +
```

Comment:
the indices in this expression do not match as there are 4 free-indices on the LHS and we're not sure how many in some terms on the RHS

Reply:
I'm sorry, I don't understand the issue. Here I tried to express the Young's tensor as Y^{aaaa}, Y^{aabb}, Y^{abab} and Y^{abba} as in Eq. 11 - 13 in [https://doi.org/10.1088/1361-6382/aa9ccc ](url).

Reply:
yes, but you are switching between Einstein summation convention and not all over the place here. It's basically impossible to know which indices are being summed over. We're asking you to stick to Einstein summation convention where any repeated indices are summed. E.g. `delta^{ij} delta^{jk} = delta^{ij}`

Reply:
I see. I meant only for pairs of lower and upper indices to be summed over. I have rewritten it to work with the convention.